Projects/Feature/OnboardingFeature/Sources/Onboarding/OnboardingCore.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  OnboardingCore.swift
+//  OnboardingFeature
+//
+//  Created by devMinseok on 7/22/24.
+//  Copyright © 2024 PomoNyang. All rights reserved.
+//
+
+import Foundation
+
+import ComposableArchitecture
+
+@Reducer
+public struct OnboardingCore {
+  @ObservableState
+  public struct State: Equatable {
+    public init() { }
+    var data = RandomAccessEquatableItems<OnboardingItem>(elements: OnboardingItemsData)
+    var fakedData: RandomAccessEquatableItems<OnboardingItem> = .init(elements: [])
+    var width: CGFloat = .zero
+    var currentIdx: Int = 0
+    var currentItemID: String = ""
+  }
+
+  public enum Action: BindableAction {
+    case onApear
+    case setIndex(Int)
+    case calculateOffset(CGFloat, OnboardingItem)
+    case dragStart
+    case _timerStart
+    case _timerEnd
+    case _timerTicked
+    case _nextPage(Int)
+    case _resetTofront
```

Comment:
이 case만 앞에 `_`를 붙인 이유가 있을까?

Reply:
만약에 구분할 필요가 있는거라면 아래처럼 하면 좋을거 같아!

```suggestion
    case timer(TimerAction)
    enum TimerAction {
      case start
      case end
      case ticked
    }
```

Reply:
아 나는 보통 뷰에서 직접 보내는 action이 아닌 '리듀서 안에서만 왔다갔다 하는 internal action'이라고 _를 붙여서 구분해줬었어 !! 타이머끼리 묶은건 아닌데 어떻게 하는게 좋을까 ~?~?

Reply:
reducer 내부에서만 쓰일땐 `_` 붙이는거 나쁘지 않은거 같아!